xpdacq/beamtimeSetup.py

```diff
@@ -231,9 +231,9 @@ def _end_beamtime(base_dir=None, archive_dir=None, bto=None):
     _end_beamtime_core(base_dir, archive_dir, bto)
     terminator = ExitAutocall()
     terminator
-    print("INFO: please type:\n"
+    print("INFO: beamtime is finished. It's IMPORTANT to type:\n"
```

Comment:
why are we asking for this confirmation?  Either we already asked for confirmation earlier to flush the directories and this is a gratuitous ask, or this is the only time we ask if they really want to end the beamtime, but it is a bit late because if they don't then we have already deconstructed the directory structure.

Let's maybe talk about this one.